docs/design/3_2_multirotor.tex

```diff
@@ -0,0 +1,230 @@
+The next tier of FC have more advanced features such as GPS-hold, autonomous flight, anto-land, 
+telemetry, etc. They are designed for more advanced operation. These FCs are more expensive costing 
+CA\$50 to CA\$200. However, these FCs have excellent after-sale support for their respective 
+manufacturers and documentation is better. The FCs are also rigorously tested in the industry. Along 
+with frequent manufacturer firmware updates, the mid-tier FCs are more liable. The downside is that 
+they are typically heavier with weather-resistent enclosures and takes up a much larger footprint. 
+Below is a list of mid-tier FCs:
+
+\begin{itemize}[noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
+    \item ArduPilot APM 2.8 is a flight controller using Arduino Mega and supports GPS and telemetry.
+    \item Pixhawk PX4 Autopilot features pre-programmable autonomous operations.
+    \item DJI Naza M Lite.
+\end{itemize}
+
+For the applications we need, we choose ArduPilot APM 2.8 as it is the most flexible option with advanced features at a reasonable price of CA\$75.00. Additionally due to of the low-cost of low-tier FCs, we will also purchase one of Naze32 or CC3D FC for back-up.
+
+\subsubsection{Radio Systems}
+
+The minimum number of channels required to fly a drone is 4. One for each control: throttle, yaw, 
+pitch, and roll. For that reason, we opt for the cheapest radio transmitter (TX) and receiver  (RX) 
+combo we find from online vendors. The weakest TX/RX combination has power emission rating of 25 mW. 
+From testing, 25 mW is more than enough to satisfy the radio requirement of 300 m.
+
+\subsubsection{Battery}
+
+We choose lithium polymer (Li-Po) batteries because they have the highest energy density (highest 
+capacity to weight ratio), and thus perfect for high-power, low-weight applications such as flying. 
+
+Li-Po batteries have mainly three design parameters: configuration, capacity, and discharge rating. 
+Configuration is how the Li-Po batteries are wired up: a single-cell Li-Po battery can provide a 
+nomial voltage of 3.7 V due to its electrochemistry. Multiple cells can be put in seriesa and the 
+voltage adds up: 2S (2 cells in series) batteries have 7.4 V output, 3S batteries have 11.1 V output,
+ and 4S batteries have 14.8 V output.
+
+Battery capacity is measured in watt-hours (Wh) or mili-amp-hours (mAh) which is power multiplied by time. A 20 Wh battery can output 20 W for 1 hour. Multiple cells of batteries can be put in parallel and the capacity adds up. 4S2P batteries have 14.8 V ouput but has double the capacity than 4S batteries.
+
+Lastly, the discharge rating is denoted by “C” and indicates the maximum discharge current. A 1,000 
+mAh battery with a discharge rating of 20C can discharge a maximum of 1.0 A * 20C = 20 A.
+
+We choose the design parameters for the battery after we have consolidated on rest of the parts for the RPAS since batteries comes in many sizes and configurations, and allows more flexibility.
```

Comment:
```suggestion
We choose the design parameters for the battery after we have consolidated on rest of the parts for the RPAS since batteries comes in many sizes and configurations, allowing for more flexibility.
```